job-templates/kubernetes_release_staging.json

```diff
@@ -43,10 +43,10 @@
       "adminUsername": "azureuser",
       "adminPassword": "replacepassword1234$",
       "sshEnabled": true,
-      "windowsPublisher": "microsoft-aks",
-      "windowsOffer": "aks-windows",
-      "windowsSku": "2019-datacenter-core-smalldisk-2008",
-      "imageVersion": "17763.1397.200820"
+      "windowsPublisher": "MicrosoftWindowsServer",
+      "windowsOffer": "WindowsServer",
+      "windowsSku": "2019-Datacenter-Core-with-Containers-smalldisk",
+      "imageVersion": "latest"
```

Comment:
Is there a specific image version we can test against?

Reply:
There is - 17763.1457.2009030514 - but part of the intentions behind setting latest was to not need to change the staging template each month.

Is there a better place for doing this?
The Sept 2020 patches came out a week ago (Sept 8) and I think it would be good to have new patches tested somewhere automatically in the future.

Reply:
SGTM